aws_list_all/listing.py

```diff
@@ -123,222 +105,42 @@ def export_resources(self, filename):
-            response['EventBuses'] = [wg for wg in response.get('EventBuses', []) if wg['Name'] != 'default']
-
-        # XRay has a "Default"  group that is always present
-        if self.service == 'xray' and self.operation == 'GetGroups':
-            response['Groups'] = [wg for wg in response.get('Groups', []) if wg['GroupName'] != 'Default']
-
-        if self.service == 'route53resolver':
-            if self.operation == 'ListResolverRules':
-                response['ResolverRules'] = [
-                    rule for rule in response.get('ResolverRules', [])
-                    if rule['Id'] != 'rslvr-autodefined-rr-internet-resolver'
-                ]
-            if self.operation == 'ListResolverRuleAssociations':
-                response['ResolverRuleAssociations'] = [
-                    rule for rule in response.get('ResolverRuleAssociations', [])
-                    if rule['ResolverRuleId'] != 'rslvr-autodefined-rr-internet-resolver'
-                ]
-
-        if 'Count' in response:
-            if 'MaxResults' in response:
-                if response['MaxResults'] <= response['Count']:
-                    complete = False
-                del response['MaxResults']
-            del response['Count']
-
-        if 'Quantity' in response:
-            if 'MaxItems' in response:
-                if response['MaxItems'] <= response['Quantity']:
-                    complete = False
-                del response['MaxItems']
-            del response['Quantity']
-
-        for neutral_thing in ('MaxItems', 'MaxResults', 'Quantity'):
-            if neutral_thing in response:
-                del response[neutral_thing]
-
-        for bad_thing in (
-            'hasMoreResults', 'IsTruncated', 'Truncated', 'HasMoreApplications', 'HasMoreDeliveryStreams',
-            'HasMoreStreams', 'NextToken', 'NextMarker', 'nextMarker', 'Marker'
-        ):
-            if bad_thing in response:
-                if response[bad_thing]:
-                    complete = False
-                del response[bad_thing]
-
-        # Special handling for Aliases in kms, there are some reserved AWS-managed aliases.
-        if self.service == 'kms' and self.operation == 'ListAliases':
-            response['Aliases'] = [
-                alias for alias in response.get('Aliases', [])
-                if not alias.get('AliasName').lower().startswith('alias/aws')
-            ]
+        complete = apply_filters(self.input, self.unfilter, response, complete)
+        unfilterList = convert_unfilterList(self.unfilter)
 
         # Special handling for service-level kms keys; derived from alias name.
-        if self.service == 'kms' and self.operation == 'ListKeys':
-            list_aliases = run_raw_listing_operation(self.service, self.region, 'ListAliases', self.profile)
+        if 'kmsListKeys' not in unfilterList and self.input.service == 'kms' and self.input.operation == 'ListKeys':
```

Comment:
I'd put a comment here that this may need rework, since it relies on other files being present.